.circleci/config.yml

```diff
@@ -153,8 +153,11 @@ commands:
       - run:
           name: Deploying to k8s cluster for service << parameters.k8s_svc_name >>
           command: |
+            git clone https://github.com/binary-com/devops-ci-scripts
+            cd ./devops-ci-scripts/k8s-build_tools
```

Comment:
Nitpick - I don't think it's healthy to CD like this, and I'm not sure why we do.

Reply:
Definitely will fix that latter